src/Apps/Sell/Components/BottomFormNavigation.tsx

```diff
@@ -1,63 +1,75 @@
-import { Box, Button, Flex, Link } from "@artsy/palette"
+import { Box, Button, Flex } from "@artsy/palette"
 import { useSellFlowContext } from "Apps/Sell/SellFlowContext"
-import { useRouter } from "System/Hooks/useRouter"
 import { useFormikContext } from "formik"
 
 export const BottomFormNavigation = () => {
-  const { isValid, isSubmitting, submitForm } = useFormikContext()
-  const { router } = useRouter()
+  return (
+    <Flex
+      width="100%"
+      p={[2, 4]}
+      flexDirection="row"
+      justifyContent="space-between"
+      alignItems="center"
+    >
+      <BottomFormBackButton />
+      <BottomFormNextButton />
```

Comment:
I'm separating both buttons to be able to render the loading spinner independently for each button when `isSubmitting` is true.